src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -137,7 +153,14 @@ void initializeReference() {
      */
     void initializeReads() {
         if (! readArguments.getReadFiles().isEmpty()) {
-            SamReaderFactory factory = SamReaderFactory.makeDefault().validationStringency(readArguments.getReadValidationStringency());
+            // Prefetcher is useful for cloud files because of the latencies involved.
+            boolean gsPathPresent = readArguments.getReadPaths().stream()
+                .filter(p->p.toUri().getScheme().equals("gs")).findAny().isPresent();
```

Comment:
In the case of no index, though, we'd need a SamReaderFactory.open(path, wrapper), which doesn't currently exist.

I need to investigate a bit more, but for some reason my IDE is suddenly unable to compile the code. I'm getting "Unable to resolve com.github.samtools:htsjdk:2.8.1-9-ged15dc0-20170110.192756-1", as if it's unable to find the artifactory.

It compiles fine from the command line, which suggests this may be a bug in the IDE.